book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] Let's add a method for adding rentals. the book itself should be sent as a parameter to create the new rental achieving the `has-many` association.